share/completions/jhipster.fish

```diff
@@ -0,0 +1,37 @@
+# JHipster 4.9.0
+
+# Check if command already given
+function __fish_prog_needs_command
+    set cmd (commandline -opc)
+    echo $cmd
+    if [ (count $cmd) -eq 1 -a $cmd[1] = 'jhipster' ]
+        return 0
+    end
+    return 1
+end
+
+# Options
+complete -f -c jhipster -n '__fish_prog_needs_command' -s d -d 'Enable debugger'
+complete -f -c jhipster -n '__fish_prog_needs_command' -s h -d 'Output usage information'
+complete -f -c jhipster -n '__fish_prog_needs_command' -s V -d 'Output version number'
+
+# Commands
+complete -f -c jhipster -n '__fish_prog_needs_command' -a app -d 'Create a new JHipster application based on the selected options.'
```

Comment:
Just `Create a new app`? (Note: I know exactly nothing about jhipster)

That it's a JHipster thing is implicit since you're executing jhipster, and that it's based on the selected options oes without saying.

Also, please no trailing ".".